modules/mod_ginger_embed/templates/embed/share-link.tpl

```diff
@@ -0,0 +1,2 @@
+<a id="{{ #embed }}" href="" title="{_ Embed _}" class="">{_ Embed _}</a>
```

Comment:
This file has the wrong path: it must be `templates/share/share-link.tpl` to be picked up by mod_ginger_base’s `share.tpl`.